src/algebra/order/monoid_lemmas.lean

```diff
@@ -132,6 +132,9 @@ lemma mul_lt_mul_of_lt_of_lt [covariant_class α α (*) (<)] [covariant_class α
 calc  a * c < a * d : mul_lt_mul_left' h₂ a
         ... < b * d : mul_lt_mul_right' h₁ d
 
+alias mul_lt_mul_of_lt_of_lt ← mul_lt_mul'''
```

Comment:
Is this necessary? I would like to get rid of lemmas with this many primes.

Reply:
We can just remove it if we replace it with `mul_lt_mul_of_lt_of_lt` in other proofs.